src/resolvers/Mutation/updateSessionTimeout.ts

```diff
@@ -0,0 +1,83 @@
+import { User } from "../../models";
+import {
+  COMMUNITY_NOT_FOUND_ERROR,
+  INVALID_TIMEOUT_RANGE,
+  USER_NOT_FOUND_ERROR,
+  UPDATE_SESSION_TIMEOUT_ARGUMENT_MISSING_ERROR,
+} from "../../constants";
+import type { MutationResolvers } from "../../types/generatedGraphQLTypes";
+import { errors, requestContext } from "../../libraries";
+import { superAdminCheck } from "../../utilities";
+import { Community } from "../../models/Community";
+
+/**
+ * Payload provided with the request for updating organization timeout.
+ */
+interface InterfaceUpdateTimeoutArgs {
+  timeout: number;
+}
+
+/**
+ * This function updates the session timeout and can only be performed by superadmin users.
+ * @param _parent - parent of the current request
+ * @param args - payload provided with the request, including organizationId and timeout
+ * @param context - context of the entire application, containing user information
+ * @returns - A message true if the organization timeout is updated successfully
+ * @throws - NotFoundError: If the user or organization is not found
+ * @throws - ValidationError: If the user is not an admin or superadmin, or if the timeout is outside the valid range
+ * @throws - InternalServerError: If there is an error updating the organization timeout
+ */
+
+export const updateSessionTimeout: MutationResolvers["updateSessionTimeout"] =
+  async (_parent, args: InterfaceUpdateTimeoutArgs, context) => {
+    if (!args.timeout) {
```

Comment:
Type safety would ensure that the timeout parameter in not nill. Thus this check is moot.

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/97614113/82bf53d6-2698-488b-b80a-f42b63f9e0f0)

Removed the the argument check.